packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## Other changes (May 20, 2025)
+- Fixed spelling mistakes in comments: changed 'Prodived' to 'Provided', 'succesful' to 'successful', and 'usages' to 'usage'. See [PR #2435](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2435).
```

Comment:
I feel like it's unnecessary to add a changelog entry if there are no changes besides code comments.

The changelogs are mostly helpful for folks to seek reasons of API / behaviour changes. We want to keep them useful and concise.